packages/grpc-web/package.json

```diff
@@ -30,7 +30,7 @@
     "google-closure-compiler": "~20200224.0.0",
     "google-closure-deps": "~20210601.0.0",
     "google-closure-library": "~20210808.0.0",
-    "google-protobuf": "~3.14.0",
+    "google-protobuf": "~3.21.1",
```

Comment:
btw, since this is only a "devDependencies" and is only used during development, it does not change the final output of the grpc-web npm package (which doesn't depend on google-protobuf directly).

What would make a difference is when developers start depending on `google-protobuf@3.21.1` directly in their client applicaiton, like in the following HelloWorld example:

https://github.com/grpc/grpc-web/blob/b0ea9c7c45d6f9ea5338fc61a81b47f589a91259/net/grpc/gateway/examples/helloworld/package.json#L10